app/models/language_tracking.rb

```diff
@@ -104,7 +104,7 @@ def periodically_clean_up
 
     @@last_clean = Time.zone.now
     glob = tag_file("*")
-    for file in Dir.glob(glob)
+    Dir.glob(glob).each do |file|
```

Comment:
RuboCop (via Codeclimate) says 

Redundant begin block detected. NEW
```ruby
 105     begin
```
Severity: MinorFound in [app/models/language_tracking.rb](https://codeclimate.com/github/MushroomObserver/mushroom-observer/app/models/language_tracking.rb?from_sha=bdbb7a48&to_sha=91e2333c) by [rubocop](https://docs.codeclimate.com/docs/rubocop)

Is that correct?
If so, what should I do? Delete both the `begin` and `each`? (Tests do pass if I do that.)

Reply:
Yeah, I probably didn't know for sure whether I needed a begin-end block around the rescue, so just threw it in to be sure.  If rubocop says it's not necessary, then I trust it.